Quantum-Control-Applications/Superconducting/Two-Fixed-Coupled-Transmons/README.md

```diff
@@ -0,0 +1,97 @@
+
+7. [Power Rabi](07_power_rabi.py) - A Rabi experiment sweeping the amplitude of the microwave pulse.
+
+8. [Ramsey Chevron](08_ramsey_chevron.py) - Conducts a 2D sweep (detuning vs idle time) to generate a Ramsey chevron pattern.
+
+9. **Readout Optimization:**
+    - [Frequency Optimization](09a_readout_optimization_freq.py) - Optimizes readout frequency by scanning frequency and calculating the signal-to-noise ratio (SNR).
+    - [Amplitude Optimization](09b_readout_optimization_amp.py) - Measures readout fidelity for different readout powers.
+    - [Duration Optimization](09c_readout_optimization_duration.py) - Analyzes SNR as a function of readout pulse duration.
+    - [Weight Optimization](09d_readout_weight_optimization.py) - Computes normalized optimal readout weights from demodulated state trajectories.
+
+10. [IQ Blobs](10_IQ_blobs.py) - Creates IQ blobs for the qubit in ground and excited states, providing parameters for active reset.
+
+11. [T1 Measurement](11_T1.py) - Measures the qubit's relaxation time T1.
+
+12. [T2 Echo Measurement](12_T2echo.py) - Measures the T2 echo time of the qubit.
+
+13. [T2 Ramsey Measurement](13_T2ramsey.py) - Measures the T2 Ramsey time using Ramsey interference.
+
+14. **Calibration:**
+    - [AC Stark Calibration](14a_ac_stark_calibration.py) - Calibrates the AC Stark shift for a qubit.
+    - [DRAG Calibration](14b_drag_calibration.py) - Performs calibration for DRAG (Derivative Removal by Adiabatic Gate).
+
+15. [All XY Experiment](15_allxy.py) - Estimates gate imperfections using the ALL XY protocol.
+
+16. **Randomized Benchmarking:**
+    - [Single Qubit Randomized Benchmarking](16a_single_qubit_RB.py) - Measures single-qubit gate fidelity through randomized benchmarking.
+    - [Interleaved Single Qubit Randomized Benchmarking](16b_single_qubit_RB_interleaved.py) - Conducts interleaved randomized benchmarking for more robust fidelity estimation.
+
+17. **Cross-Resonance Calibration:**
+    - [CR Time Rabi with 1Q QST](17a_CR_time_rabi_1q_QST.py) - Measures QST on the target qubit by applying CR drive on the control qubit.
+    - [Cancel CR Time Rabi with 1Q QST](17b_cancelCR_time_rabi_1q_QST.py) - Measures QST on the target qubit by applying CR drive on the control qubit along with cancel drive on the target qubit.
+    - [Echoed CR Time Rabi with 1Q QST](17c_echoCR_time_rabi_1q_QST.py) - Measures QST on the target qubit by applying echoed CR drive on the control qubit along with cancel drive on the target qubit.
+
+18. **Hamiltonian Tomography:**
+    - [Unit Hamiltonian Tomography](18a_CR_calib_unit_hamiltonian_tomography.py) - Calibrates CR gates by estimating Hamiltonian parameters through state tomography.
+    - [CR Drive Amplitude Calibration](18b_CR_calib_cr_drive_amplitude.py) - Calibrates the amplitude of the CR drive.
+    - [CR Drive Phase Calibration](18c_CR_calib_cr_drive_phase.py) - Calibrates the phase of the CR drive.
```

Comment:
Actually here we calibrate the phase of the CR drive and the cancel drive no?

Reply:
Yes, fixed it.